src/OrchardCore.Modules/OrchardCore.Resources/Startup.cs

```diff
@@ -23,6 +24,8 @@ public override void ConfigureServices(IServiceCollection serviceCollection)
             });
 
             serviceCollection.AddTransient<IConfigureOptions<ResourceManagementOptions>, ResourceManagementOptionsConfiguration>();
+
+            serviceCollection.TryAddTransient<IConfigureOptions<ResourceOptions>, LocalResourceConfigureOptions>();
```

Comment:
Scoped is fine as  Setting won't change for each requested instance

Reply:
Same as above. I don’t think this is permitted